scripts/complete_liquidity_provision.js

```diff
@@ -105,15 +116,29 @@ module.exports = async (callback) => {
       callback("Error: Choose a smaller fleetSize, otherwise your payload will be to big for Infura nodes")
     }
 
-    console.log(`2. Deploying ${argv.fleetSize} trading brackets`)
-    const bracketAddresses = await deployFleetOfSafes(masterSafe.address, argv.fleetSize, true)
-    console.log("List of bracket traders in one line:", bracketAddresses.join())
+    let bracketAddresses
+    if (argv.brackets.length > 0) {
+      console.log("==> Skipping safe deployment and using brackets safeOwners")
+      bracketAddresses = argv.brackets
+      // Ensure that safes are all owned solely by masterSafe
+      for (const safeAddr of argv.brackets) {
+        const safeOwners = await (await GnosisSafe.at(safeAddr)).getOwners()
```

Comment:
Can we use the existing masterSafe var instead of calling .at again?

Reply:
For the actual checks, we have already code for it:

```
assert(await isOnlySafeOwner(masterAddress, bracketAddress), "each bracket should be owned only by the master Safe")

        assert(await checkNoOrdersPlaced(bracketAddress, exchange), "each bracket should be owned only by the master Safe")
```
Take also a look into this PR:
https://github.com/gnosis/dex-liquidity-provision/pull/196/files#

Here Federico requests to put the orderchecks into this PR.

Reply:
> Can we use the existing masterSafe var instead of calling .at again?

These are not master safes. They are the slave safes.

Reply:
@josojo are you suggesting that I wait until #196 is merged? Or include this suggestion as an addition here? I imagine it wouldn't hurt to ensure no orders exist for these brackets, but I really don't like this strict enforcement. What if I wanted to reuse the same brackets with renewed orders? I could make a "proceedAnyway" verification, but first please clarify what it was I was I am supposed to use from #196 or if I am to wait until it is merged.

Reply:
Every safe is new, so you need to await it at some point. As it is right now it would be very slow however because for 20 safes you'd be sending 40 requests in series to Infura, so everything should be wrapped at least by a `Promise.all()`.
I'd consider simply calling `verifyBracketsWellFormed` from `verify_script.js` and not add new ad hoc checks here.

Reply:
Yes incorporating the ordercheck from PR #196 is the best way in my opinion.

Additionally, you can include the whole function : `verifyBracketsWellFormed` then we should be quite safe.

> but I really don't like this strict enforcement. 
fair enough, a prompt would be the best solution for sure

Reply:
Does this `verifyBracketsWellFormed` already exist and does it do both of the checks we would like to have here? 

That is, that each safe is solely owned by master and that there are not yet any orders?

My new problem with this is that having both ownership and non-existent orders tied up in the same constraint means that I cannot reuse safes for different orders. I would prefer if we have a hard constraint on ownership (because we must) but have an option to "proceed anyway" for the orders check. 

Any opinions?

Reply:
@josojo - Could we merge this as is (i.e. without the check for orders) or can we get the order check merged beforehand. I feel like this would be an easy fix/feature to include once both of these are merged rather than have this hanging blocked by the pending PR.